<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sylfaen</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>VENTANA PRINCIPAL</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_product">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>431</width>
      <height>351</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>CÓDIGO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NOMBRE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PRECIO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CATEGORÍA</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_listar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 255, 187);
selection-color: rgb(28, 255, 187);
border-color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>LISTAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_update">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 255, 187);
selection-color: rgb(28, 255, 187);
border-color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>ACTUALIZAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_create">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 255, 187);
selection-color: rgb(28, 255, 187);
border-color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>CREAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_read">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>490</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 255, 187);
selection-color: rgb(28, 255, 187);
border-color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>SELECCIONAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>490</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 144, 150);
selection-color: rgb(255, 144, 150);
border-color: rgb(255, 0, 34);</string>
    </property>
    <property name="text">
     <string>ELIMINAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
